Example/AppDelegate.swift

```diff
@@ -1,14 +1,9 @@
 import Cocoa
 import Preferences
 
-extension NSToolbarItem.Identifier {
-	static var preferenceGeneral: NSToolbarItem.Identifier {
-		return NSToolbarItem.Identifier(rawValue: "preferenceGeneral")
-	}
-
-	static var preferenceAdvanced: NSToolbarItem.Identifier {
-		return NSToolbarItem.Identifier(rawValue: "preferenceAdvanced")
-	}
+extension PreferenceIdentifier {
+	static let general = PreferenceIdentifier(rawValue: "preferenceGeneral")
```

Comment:
Can't the raw value `preferenceGeneral` just be `general`? It's already clear from the context what it is.